src/main/scala/cognite/spark/v1/FileContentRelation.scala

```diff
@@ -114,7 +118,15 @@ class FileContentRelation(config: RelationConfig, fileExternalId: String, inferS
             byteStream
               .through(enforceSizeLimit)
               .through(fs2.text.utf8.decode)
-              .through(fs2.text.lines)
+              .through(fs2.text.linesLimited(lineSizeLimitCharacters))
+              .handleErrorWith {
```

Comment:
there's `.zipWithIndex` method that can expose indexes, not sure if we can attach it to error message though :thinking: 
But that's only a nice to have extra error detail

Reply:
It is possible, but then we cannot use linesLimited, we'd need to do a second pass with indexes and limit there. Not very hard, how much do you think this is important?